packages/discovery-react-components/src/typings.d.ts

```diff
@@ -20,7 +20,7 @@ declare module '@carbon/icons-react/*';
 declare module '*.md';
 declare module 'marked';
 declare module 'uuid';
-declare module 'carbon-components-react/lib/components/ListBox';
+declare module 'carbon-components-react/es/components/ListBox';
```

Comment:
I think the reason this is here is due to including this specific component in `rollup.config.js`. I don't remember why that was added there. That ref should be changed to `/es/` as well.

Reply:
Oh, I see: we're using `ListBox` but it's not exported at the "top level" by carbon components.

Reply:
Right. I'm not very familiar with Typescript but assumed this needed to be updated as well when I updated the path for the ListBox.